Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -14,16 +14,78 @@ static void Main(string[] args)
             Console.ReadKey(true);
         }
 
-        private static void Test(string args, int expected)
-        {
+        private static void Test(string args, int expected)
+        {
             var result = FindDigit(args).Equals(expected) ? "PASS" : "FAIL";
-            Console.WriteLine($"{args} : {result}");
+            Console.WriteLine($"{args} : {result}");
         }
 
-        public static int FindDigit(string equation)
-        {
-            // Add your code here.
-            throw new NotImplementedException();
+        public static int FindDigit(string equation)
+        {
+            var str = equation.Split("=");
+            var c = str[1];
+            str = str[0].Split('*');
+            var a = str[0];
+            var b = str[1];
+
+            var res = "";
+            if (c.IndexOf('?') != -1) {

```

Comment:
This is like you are coding in `C`. I see very similar code going down. Can we have a method here instead of the procedural code?